msg/CartesianTrajectoryPoint.msg

```diff
@@ -0,0 +1,14 @@
+geometry_msgs/Pose pose
+geometry_msgs/Twist velocity
+geometry_msgs/Accel acceleration
+
+geometry_msgs/Twist pose_upper_tolerance
+geometry_msgs/Twist pose_lower_tolerance
```

Comment:
I may be mathematically off here, but I think a tolerance for a pose is not a "twist"

Descartes simply defines tolerances as (x y z), (r p y)

https://github.com/ros-industrial-consortium/descartes/blob/kinetic-devel/descartes_trajectory/include/descartes_trajectory/cart_trajectory_pt.h

@mlautman thoughts?

Reply:
True, there is no time involved here.